src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -109,7 +109,9 @@ public Socket(AddressFamily addressFamily, SocketType socketType, ProtocolType p
             _socketType = socketType;
             _protocolType = protocolType;
 
-            Debug.Assert(addressFamily != AddressFamily.InterNetworkV6 || !DualMode);
+            // TODO: I am not hitting this assertion in the desktop code,
```

Comment:
@CIPop This was the assertion I was hitting that I mentioned to you earlier. I was hitting it on CoreCLR on Windows, but not on the desktop framework. I haven't checked to see if I hit on OSX/Linux, but I think we will want to take a look at it regardless to see if it's a valid assertion.